src/main/java/roomescape/controller/AdminViewController.java

```diff
@@ -0,0 +1,18 @@
+package roomescape.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class AdminViewController {
+    @GetMapping(value = "/")
```

Comment:
```suggestion
public class AdminViewController {

    @GetMapping(value = "/")
```
저희 동아리 코드컨벤션은 클래스 시작 첫 줄은 개행을 해줍니다!